spec/controllers/admin/payments_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe Admin::PaymentsController do
+  describe "get :index" do
+    it "sets @payments to all the payments" do
+      2.times { Fabricate(:payment) }
+      set_admin
+      get :index
+      expect(assigns(:payments)).to be_present
```

Comment:
This verifies that `@payments` is set, but doesn't verify the second part of your description "to all the payments".